sympy/tensor/array/expressions/tests/test_from_to_einsum.py

```diff
@@ -0,0 +1,55 @@
+import random
+import string
+from collections import Counter
+
+from sympy import Array
+from sympy.external import import_module
+from sympy.tensor.array.expressions.from_to_einsum import einsum_to_sympy_array
+
+numpy = import_module("numpy")
+
+
+def _compare_einsum_numpy(path, *a):
+    sa = [Array(i) for i in a]
+    assert (numpy.einsum(path, *a) == numpy.array(einsum_to_sympy_array(path, *sa).as_explicit().tolist())).all()
+
+
+def _get_random_path(*a):
```

Comment:
This doesn't need to be done here, but this is exactly the sort of thing I'd like to see [hypothesis](https://hypothesis.readthedocs.io/en/latest/index.html) used for (https://github.com/sympy/sympy/issues/20914)